blocks/test/full-content.js

```diff
@@ -95,13 +95,23 @@ describe( 'full post content fixture', () => {
 
 	fileBasenames.forEach( f => {
 		it( f, () => {
+			/**
```

Comment:
Mine is `//`, but I also didn't realize 'til recently that this was at odds with the core convention 😬  I'd say I have a preference _against_ this style, only because it is easy to confuse with ([and encounter errors as](http://eslint.org/docs/rules/valid-jsdoc)) JSDoc.